integration-core/src/main/java/org/symphonyoss/integration/core/bridge/IntegrationBridgeExceptionHandler.java

```diff
@@ -146,7 +169,7 @@ private void updateStreams(IntegrationInstance instance, String integrationUser,
       removeStreamFromInstance(instance, integrationUser, stream);
       notifyInstanceOwner(instance, integrationUser, roomName);
     } catch (IntegrationRuntimeException | IOException e) {
-      LOGGER.error("Fail to update streams", e);
+      LOGGER.error(logMessage.getMessage(FAIL_UPDATE_STREAM) ,e);
```

Comment:
Where's the solution? IntegrationRuntimeException should have the message and solution.